apps/dot_ext/models.py

```diff
@@ -343,4 +403,80 @@ def get_application_require_demographic_scopes_count():
     return None
 
 
-post_delete.connect(archive_token, sender='oauth2_provider.AccessToken')
+def get_access_token_counts():
```

Comment:
START-CODE-FOR-REVIEW-SECTION

Creates `get_access_token_counts()` and `get_access_token_by_app_counts(application)` functions that return access_token related counts.